packages/block-library/src/post-title/edit.js

```diff
@@ -58,20 +58,28 @@ export default function PostTitleEdit( {
 	const { title, link } = post;
 
 	let titleElement = (
-		<PlainText
-			tagName={ TagName }
-			placeholder={ __( 'No Title' ) }
-			value={ title }
-			onChange={ ( value ) =>
-				editEntityRecord( 'postType', postType, postId, {
-					title: value,
-				} )
-			}
-			__experimentalVersion={ 2 }
-			{ ...( isLink ? {} : blockProps ) }
-		/>
+		<TagName { ...( isLink ? {} : blockProps ) }>
+			{ __( 'Placeholder for post title' ) }
```

Comment:
I wonder if there's a better way to convey this... Maybe something like:

`Post title (placeholder)`

I also wonder if we can contextualize this placeholder to the actual `postType`; So if I'm editing a Page it would say `Page title (placeholder)`.

Reply:
What do you think about what @jameskoster was proposing in his proposed issue linked above - "An example title"

Reply:
I think it may be worth spending some more time exploring designs for this, and accounting for all different block types. I wouldn't say it needs to hold up this PR.